pallets/creditcoin/src/tests.rs

```diff
@@ -2225,10 +2230,58 @@ fn exempt_should_succeed() {
 
 		// assert events in reversed order
 		let mut all_events = <frame_system::Pallet<Test>>::events();
+
+
+		log::debug!("===== ALL EVENTS={:?}", all_events.len());
+		for ev in all_events.iter() {
+			log::debug!("==== event={:?}", ev.event);
+		}
```

Comment:
@nathanwhit  I don't see the `balances.Withdraw` event here. Here's all of them:

```
test tests::exempt_should_succeed ... Mar 18 11:38:42.786 DEBUG pallet_creditcoin::tests: ===== ALL EVENTS=8    
Mar 18 11:38:42.786 DEBUG pallet_creditcoin::tests: ==== event=Event::Creditcoin(Event::AddressRegistered(AddressId(0xfd3fc544c39c813a7c34ffe1197f443fb327505d174b90a44fb5bd1e2eda792e), Address { blockchain: Blockchain::Rinkeby, value: BoundedVec([48], 256), owner: 0000000000000000000000000000000000000000000000000000000000000000 (5C4hrfjw...) }))    
Mar 18 11:38:42.787 DEBUG pallet_creditcoin::tests: ==== event=Event::Creditcoin(Event::AddressRegistered(AddressId(0xba2479a7462fcd2687d2bf0b477722f8e180177e29dce1a713222e0f6f6f7694), Address { blockchain: Blockchain::Rinkeby, value: BoundedVec([49], 256), owner: 0101010101010101010101010101010101010101010101010101010101010101 (5C62Ck4U...) }))    
Mar 18 11:38:42.787 DEBUG pallet_creditcoin::tests: ==== event=Event::Creditcoin(Event::AskOrderAdded(AskOrderId(1000, 0x74285904af79ec6d4bc1f3161c553a3eabe943c363d17af6c4bcb3e48d30c5ab), AskOrder { blockchain: Blockchain::Rinkeby, lender_address_id: AddressId(0xfd3fc544c39c813a7c34ffe1197f443fb327505d174b90a44fb5bd1e2eda792e), terms: AskTerms(LoanTerms { amount: 10000000, interest_rate: 1, maturity: 1000000 }), expiration_block: 1000, block: 1, lender: 0000000000000000000000000000000000000000000000000000000000000000 (5C4hrfjw...) }))    
Mar 18 11:38:42.787 DEBUG pallet_creditcoin::tests: ==== event=Event::Creditcoin(Event::BidOrderAdded(BidOrderId(1000, 0xe913f1797e640c44d8d5119af888a7e6df5125c72729393fca9abdbdd9ef0e3f), BidOrder { blockchain: Blockchain::Rinkeby, borrower_address_id: AddressId(0xba2479a7462fcd2687d2bf0b477722f8e180177e29dce1a713222e0f6f6f7694), terms: BidTerms(LoanTerms { amount: 10000000, interest_rate: 1, maturity: 1000000 }), expiration_block: 1000, block: 1, borrower: 0101010101010101010101010101010101010101010101010101010101010101 (5C62Ck4U...) }))    
Mar 18 11:38:42.787 DEBUG pallet_creditcoin::tests: ==== event=Event::Creditcoin(Event::OfferAdded(OfferId(1000, 0x2c94cb575857e41a489db05ad951206f2b40700dd2665791ada288333e7740d2), Offer { blockchain: Blockchain::Rinkeby, ask_id: AskOrderId(1000, 0x74285904af79ec6d4bc1f3161c553a3eabe943c363d17af6c4bcb3e48d30c5ab), bid_id: BidOrderId(1000, 0xe913f1797e640c44d8d5119af888a7e6df5125c72729393fca9abdbdd9ef0e3f), expiration_block: 1000, block: 1, lender: 0000000000000000000000000000000000000000000000000000000000000000 (5C4hrfjw...) }))    
Mar 18 11:38:42.787 DEBUG pallet_creditcoin::tests: ==== event=Event::Creditcoin(Event::DealOrderAdded(DealOrderId(1000, 0x1f0558d0cc26acc934fedcc969cf386e48d91cf22783b76a198bde74aa004ef8), DealOrder { blockchain: Blockchain::Rinkeby, offer_id: OfferId(1000, 0x2c94cb575857e41a489db05ad951206f2b40700dd2665791ada288333e7740d2), lender_address_id: AddressId(0xfd3fc544c39c813a7c34ffe1197f443fb327505d174b90a44fb5bd1e2eda792e), borrower_address_id: AddressId(0xba2479a7462fcd2687d2bf0b477722f8e180177e29dce1a713222e0f6f6f7694), terms: LoanTerms { amount: 10000000, interest_rate: 1, maturity: 1000000 }, expiration_block: 1000, timestamp: 0, funding_transfer_id: None, repayment_transfer_id: None, lock: None, borrower: 0101010101010101010101010101010101010101010101010101010101010101 (5C62Ck4U...) }))    
Mar 18 11:38:42.788 DEBUG pallet_creditcoin::tests: ==== event=Event::Creditcoin(Event::TransferRegistered(TransferId(0xfd3fc544c39c813a7c34ffe1197f443fb327505d174b90a44fb5bd1e2eda792e), Transfer { blockchain: Blockchain::Rinkeby, kind: TransferKind::Ethless(BoundedVec([48, 120, 48, 97, 100, 49, 52, 51, 57, 97, 48, 101, 48, 98, 102, 100, 99, 100, 52, 57, 57, 51, 57, 102, 57, 55, 50, 50, 56, 54, 54, 54, 53, 49, 97, 52, 97, 97, 57, 98, 51, 99], 256)), from: AddressId(0xfd3fc544c39c813a7c34ffe1197f443fb327505d174b90a44fb5bd1e2eda792e), to: AddressId(0xba2479a7462fcd2687d2bf0b477722f8e180177e29dce1a713222e0f6f6f7694), order_id: OrderId::Deal(DealOrderId(1000, 0x1f0558d0cc26acc934fedcc969cf386e48d91cf22783b76a198bde74aa004ef8)), amount: 0, tx: BoundedVec([48], 256), block: 1, processed: false, sighash: 0000000000000000000000000000000000000000000000000000000000000000 (5C4hrfjw...) }))    
Mar 18 11:38:42.788 DEBUG pallet_creditcoin::tests: ==== event=Event::Creditcoin(Event::LoanExempted(DealOrderId(1000, 0x1f0558d0cc26acc934fedcc969cf386e48d91cf22783b76a198bde74aa004ef8), TransferId(0xfd3fc544c39c813a7c34ffe1197f443fb327505d174b90a44fb5bd1e2eda792e)))    
ok
```

All events are `Event::Creditcoin`. Is some configuration missing or these events get filtered out somehow?